pkg/util/file_indexer_test.go

```diff
@@ -243,3 +245,736 @@ func TestGenerateNewFileDataEntry(t *testing.T) {
 		})
 	}
 }
+
+func Test_recursive(t *testing.T) {
+	fs := filesystem.DefaultFs{}
```

Comment:
why not consider using the memory mapped Fs here? afero one.

Reply:
The function currently uses `filepath.Glob()` which uses `os.Lstat()` internally. The library `afero` doesn't provide an implementation for it as of now. Thus using the DefaultFs for now.

Reply:
Cool